backend/testfiles/execution/stdlib/parser.dark

```diff
@@ -322,13 +331,13 @@ module TextToTextRoundtripping =
     ("(1L, \"hello\", 2L, true)" |> roundtripCliScript) = "(1L, \"hello\", 2L, true)"
     ("(1L, 2L + 3L, 4L)" |> roundtripCliScript) = "(1L, (2L) + (3L), 4L)"
 
-    // record literal
-    ("Person {name =\"John\"} " |> roundtripCliScript) = "Person { name = \"John\" }"
-    ("Person {name =\"John\"; age = 30L} " |> roundtripCliScript) = "Person { name = \"John\"; age = 30L }"
-    ("Person {name =\"John\"; age = 30L; hasPet = true} " |> roundtripCliScript) = "Person { name = \"John\"; age = 30L; hasPet = true }"
+    //     // record literal
+    //     ("Person {name =\"John\"} " |> roundtripCliScript) = "Person { name = \"John\" }"
+    //     ("Person {name =\"John\"; age = 30L} " |> roundtripCliScript) = "Person { name = \"John\"; age = 30L }"
+    //     ("Person {name =\"John\"; age = 30L; hasPet = true} " |> roundtripCliScript) = "Person { name = \"John\"; age = 30L; hasPet = true }"
```

Comment:
This got tripped up, I think because the Person type doesn't exist. So let's use a real type, that we know will be in the PM for a bit. Also let's make sure there's an enum equivalent to this test.